source/adapters/cuda/kernel.cpp

```diff
@@ -190,12 +190,13 @@ UR_APIEXPORT ur_result_t UR_APICALL urKernelGetNativeHandle(
 }
 
 UR_APIEXPORT ur_result_t UR_APICALL urKernelSuggestMaxCooperativeGroupCountExp(
-    ur_kernel_handle_t hKernel, size_t localWorkSize,
-    size_t dynamicSharedMemorySize, uint32_t *pGroupCountRet) {
+    ur_kernel_handle_t hKernel, ur_device_handle_t hDevice,
+    size_t localWorkSize, size_t dynamicSharedMemorySize,
+    uint32_t *pGroupCountRet) {
   UR_ASSERT(hKernel, UR_RESULT_ERROR_INVALID_KERNEL);
 
-  // We need to set the active current device for this kernel explicitly here,
-  // because the occupancy querying API does not take device parameter.
```

Comment:
This comment was targeted towards the Cuda driver occupancy query API requiring to set the current device in use, so I think it's still relevant in that sense. Perhaps, it just wasn't worded very clear.

Reply:
Ok, that makes sense. I'm leaving the comment then (unless you'd like to suggest some better wording).

Reply:
Nice. Thanks for that! I think it's fine to leave it as is.